### PR TITLE
[#2PlaysAmonth] Text analyzer

### LINKED ISSUE #924: [#2PlaysAMonth]: Text Analyzer
### What's the Play about?

A Adv text analyzer that will help you  analyze your text and tell to do certain modifications and error correction and  text extraction from images and  a lot more.

### What ReactJS concept will be used to create this play?

Hooks  (will add more in the process of building it)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by yung-coder**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

A Text-Analyzer where  you will be able to analyze your text and make it more effective with features like text extraction from image and a lot more.

Fixes #924 

# How Has This Been Tested ?

It has been tested by me in my local environment 

# Checklist:

- [✅] I have performed a self-review of my own code
- [✅] I have commented my code, particularly in hard-to-understand areas
- [✅] I have made corresponding changes to the documentation
- [✅] My changes generate no new warnings
- [✅] I have added tests that prove my fix is effective or that my feature works
- [✅] New and existing unit tests pass locally with my changes
- [✅] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output
none

### CODE REVIEW

File: src/plays/text-analyzer/styles/Button.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/* CSS */
+.button-58 {
```
Comment (atapas) [inline src/plays/text-analyzer/styles/Button.css:2]: Like how you have done in your past pays, add unique style name as prefix for all your styles in all CSS files.

**Review by atapas — CHANGES_REQUESTED** [review]
@yung-coder Review comment...

also please don;t forget to record a demo on stackstream and add to your play by editing it.

File: src/plays/text-analyzer/styles/Button.css (hunk 1 above)
Comment (yung-coder) [reply src/plays/text-analyzer/styles/Button.css:2]: okay

**Comment by yung-coder** [comment]
@atapas  made the changes

**Comment by yung-coder** [comment]
Hey @atapas is it all good or need to make some changes ..?

**Comment by atapas** [comment]
@yung-coder Hello.. Your PR is also merge ready, but missing the link to the recorded demo. Please add it by editing the play so that we can merge it.

**Comment by yung-coder** [comment]
> @yung-coder Hello.. Your PR is also merge ready, but missing the link to the recorded demo. Please add it by editing the play so that we can merge it.

Sure will do it asap

**Comment by atapas** [comment]
> Visit Preview

@yung-coder Can you please take care asap?

**Comment by yung-coder** [comment]
> > Visit Preview
> 
> @yung-coder Can you please take care asap?

@atapas Sorry for the delay,  made the changes added videos to both of my plays i think one is without audio if so  i will make it again ...

**Comment by atapas** [comment]
> > > Visit Preview
> > 
> > 
> > @yung-coder Can you please take care asap?
> 
> @atapas Sorry for the delay, made the changes added videos to both of my plays i think one is without audio if so i will make it again ...

Where @yung-coder ? I do not see the video added..

image

Please edit the play and add.. Need to merge this.

**Comment by yung-coder** [comment]
> > > > Visit Preview
> > > 
> > > 
> > > @yung-coder Can you please take care asap?
> > 
> > 
> > @atapas Sorry for the delay, made the changes added videos to both of my plays i think one is without audio if so i will make it again ...
> 
> Where @yung-coder ? I do not see the video added..
> 
> image
> 
> Please edit the play and add.. Need to merge this.

@atapas made the changes added video links for both the plays please check it out...

**Review by atapas — APPROVED** [review]

**Pull request merged**